student.rb

```diff
@@ -0,0 +1,16 @@
+require './person'
+
+# Student class
+# The Student class represents a student in a school.
+# The student has a name, age, and classroom.
+# The student inherits from Person class.
+class Student < Person
+  def initialize(name, age, classroom)
```

Comment:
- [ ] please make sure to also add the parent_permission in the initialize method and pass it to super as well.

Reply:
Resolved as advised